pkg/api/server/v1alpha2/plugin/plugin_logs.go

```diff
@@ -304,10 +306,9 @@ func getBlobLogs(s *LogServer, writer io.Writer, parent string, rec *db.Record)
 		err := errors.New("pipelinerun not supported, please use taskrun")
 		s.logger.Error(err)
 		return err
-	case typeTaskRun:
-		logRec := &db.Record{}
-		if legacy {
-			logRec, err = getLogRecord(s.db, parent, rec.ResultID, rec.Name)
+	case typeTaskRunV1Beta1:
+		if legacy && rec.Type == typeTaskRunV1Beta1 {
```

Comment:
Is `&& rec.Type == typeTaskRunV1Beta1` really required? Isn't that ensured by `case typeTaskRunV1Beta1`?

Reply:
It's not required, I forgot to remove this.

Reply:
Fixed this.